<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-Attribute </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="onSelection_chb">
      <property name="text">
       <string>Only On Selection</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_le">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ByType:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type_cmb"/>
      </item>
      <item>
       <widget class="QPushButton" name="select_btn">
        <property name="text">
         <string>Select Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Attribute:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="attribute_le"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="value_le"/>
      </item>
      <item>
       <widget class="QCheckBox" name="override_chb">
        <property name="text">
         <string>Override</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectMatch_btn">
        <property name="text">
         <string>Select Match</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="apply_btn">
        <property name="text">
         <string>APPLY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lock_btn">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Lock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unlock_btn">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>UnLock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_btn">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>Remove Override</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>One or More Nodes:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connect_btn">
        <property name="text">
         <string>CONNECT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="setFrom_btn">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addFrom_btn">
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeFrom_btn">
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Objects In List:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fromListCount_lbl">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Attribute:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromAttr_le"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="removeTo_btn">
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addTo_btn">
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="setTo_btn">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Objects In List:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="toListCount_lbl">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QComboBox" name="listToCreate_cmb"/>
          </item>
          <item>
           <widget class="QPushButton" name="createForList_btn">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Attribute:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toAttr_le"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QListWidget" name="from_lw"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&gt;&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="to_lw"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Constraints</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="maintainOffset_chb">
        <property name="text">
         <string>Maintain Offset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="parent_btn">
        <property name="text">
         <string>PARENT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="point_btn">
        <property name="text">
         <string>POINT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="orient_btn">
        <property name="text">
         <string>ORIENT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scale_btn">
        <property name="text">
         <string>SCALE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
